libbeat/kibana/index_pattern_generator.go

```diff
@@ -176,9 +177,5 @@ func createTargetDir(baseDir string, version common.Version) string {
 }
 
 func getVersionPath(version common.Version) string {
-	versionPath := "6"
-	if version.Major == 5 {
-		versionPath = "5"
-	}
-	return versionPath
+	return strconv.Itoa(version.Major)
```

Comment:
@ruflin tests are failing because of this change, according to tests, path should be `6/index-pattern` for all versions major to 5, is this correct? should we continue using the generated index patterns for version 6?

Reply:
Good question. Today I would say yes but as far as I heard the structure on the Kibana side might change, so not sure if we need to adapt it. @skh is currently looking into these structure changes from the dashboard perspective.

To move this forward I would say for now lets stick to just 6.

Reply:
Ok, I revert the change here and remove generation for 7.0 by now.